Sources/MapboxCoreNavigation/NavigationService.swift

```diff
@@ -476,13 +476,15 @@ public class MapboxNavigationService: NSObject, NavigationService {
      Intializes a new `NavigationService` for replaying a session from provided `History`.
      
      - parameter history: `History` object, containing initial route and location trace to be replayed.
+     - parameter customHistoryEventsListener: Custom `ReplayManagerHistoryEventsListener` which will be used to handle replay events. Default value (`nil`)  will also loop route assignment events to update the `Router`.
```

Comment:
'replay' word is mentioned several times in the description, explaining that it does replay a history file. I don't feel that some more context on 'replay events' is needed.
'custom' suffix is added here to accompany `customRoutingProvider` we already had here, because it shares similar logic: if user provides a value - that 'custom' value will be used. Otherwise - there is a default one assumed.